frontend/test/metabase/scenarios/downloads/reproductions/21830-slow-card-actions.cy.spec.js

```diff
@@ -0,0 +1,29 @@
+import {
+  editDashboard,
+  getDashboardCard,
+  restore,
+  showDashboardCardActions,
+  visitDashboard,
+} from "__support__/e2e/cypress";
+
+describe("issue 21830", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+    cy.intercept("POST", "/api/dashboard/*/dashcard/*/card/*/query", {
+      delay: 10000,
+    });
+  });
+
+  it("should hide card actions when the card is loading (metabase#21830)", () => {
+    visitDashboard(1);
```

Comment:
This would never work because `visitDashboard` helper automatically waits for the dashcardQuery response. We need to make sure the assertion happens before that response comes back.

Reply:
Fixed, please have a look